src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -110,6 +110,12 @@ export interface StripeUPEClient {
      */
     confirmPayment(options: StripeConfirmPaymentData): Promise<{paymentIntent?: PaymentIntent; error?: StripeError}>;
 
+    /**
+     * When called, confirmPayment will attempt to complete any required actions,
```

Comment:
`confirmPayment`, you mean `confirmCardPayment`

Reply:
Updated the descriptions